apps/nerves_hub_www/lib/nerves_hub_www_web/templates/layout/_navigation.html.eex

```diff
@@ -12,7 +12,7 @@
           </a>
 
           <div class="dropdown-menu workspace-dropdown" aria-labelledby="navbarDropdownMenuLink">
-            <div class="help-text mt-3 pl-3 pr-3">Select a workspace</div>
+            <div class="help-text mt-3 pl-3 pr-3">Select organization</div>
```

Comment:
Should this be Select an Organization?

Reply:
I couldn't decide which sounded better! Looking again I think you're right